### PR TITLE
Add Ghostnet && Fix object destruction order in APIToModel function to avoid undefined behavior 

### PR DESCRIPTION
**Pull request by Phoenix8215**
- add new model——ghostnetV1&ghostnetV2
- fixed the order of object destruction in mobileNet. The original code destroyed the engine before the builder, leading to undefined behavior. The destruction order is now corrected by first destroying the engine, followed by the builder.🥰

### CODE REVIEW

**Comment by wang-xinyu** [comment]
@Phoenix8215 
Pls fix the pre-commit errors, you can refer to https://github.com/wang-xinyu/tensorrtx/blob/master/tutorials/contribution.md 

Thanks for you contribution!

**Comment by Phoenix8215** [comment]
Hi @wang-xinyu,

I have resolved the pre-commit errors as requested. You can review the latest commits to verify the changes.

Thank you for your guidance and support!💞

File: mobilenet/mobilenetv2/mobilenet_v2.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 import argparse
 
 import numpy as np
-import pycuda.autoinit
```
Comment (wang-xinyu) [inline mobilenet/mobilenetv2/mobilenet_v2.py:7]: This line can't be deleted. Add a noqa marker.

```
import pycuda.autoinit  # noqa: F401
```

File: mobilenet/mobilenetv3/mobilenet_v3.py [hunk 2]
```diff
@@ -4,7 +4,6 @@
 import argparse
 
 import numpy as np
-import pycuda.autoinit
```
Comment (wang-xinyu) [inline mobilenet/mobilenetv3/mobilenet_v3.py:7]: This line can't be deleted. Add a noqa marker.

```
import pycuda.autoinit  # noqa: F401
```

**Review by wang-xinyu — CHANGES_REQUESTED** [review]

File: mobilenet/mobilenetv3/mobilenet_v3.py (hunk 2 above)
Comment (Phoenix8215) [reply mobilenet/mobilenetv3/mobilenet_v3.py:7]: Apologies for the confusion earlier. Thank you for your patience!

**Pull request merged**